docs/installation.md

```diff
@@ -1,68 +1,68 @@
-*  downloading the artifacts and installing using the archive
+*  Using the Eclipse Marketplace
+*  Configuring an update site
+*  Downloading the artifacts and installing them by using the archive
 
 ## Using Eclipse Marketplace
 
 1. Point your browser to the Liberty Tools marketplace entry: https://marketplace.eclipse.org/content/liberty-tools
-2. Drag the "Install" button to the toolbar of your Eclipse IDE
+2. Drag the **Install** button to the toolbar of your Eclipse IDE
 
-## Using Eclipse Marketplace client
+## Using the Eclipse Marketplace client
 
-1. Go to menu:  `Help`->`Eclipse Marketplace` and type **"Liberty Tools"**.
+1. From the Eclipse menu in your IDE, select **Help** > **Eclipse Marketplace** and type **"Liberty Tools"**.
 
     **NOTE:** Do NOT select the "IBM Liberty Developer Tools" selection, which is an earlier, different set of IDE features/plugins, though with the similar name.
 
 ## Using Help -> Install New Software
 
-The Liberty Tools feature can be installed using the artifacts provided in the form of an update site or through an archive.
+You can use the **Install New Software** option in the Eclipse **Help** menu to install Liberty Tools by using an update site or through an archive.
 
-1. Go to menu: `Help`->`Install New Software`.
+1. From the Eclipse menu in your IDE, select **Help** > **Install New Software**.
 
 ![Step 1. New software installation](images/install-installNewSotwareEntry.png)
 
-2. For a given release selected from the [Releases](https://github.com/OpenLiberty/liberty-tools-eclipse/releases) list, the installation artifacts can be added in one of two ways (the specific release page shows the URLs for each):
+2. For any release in the [Releases](https://github.com/OpenLiberty/liberty-tools-eclipse/releases) list, you can add the installation artifacts either from an update site or from a downloadable archive. The entry for each release page includes the URLs for both options.
     
-    a. **UPDATE SITE** 
+    a. Install by using an update site URL. 
 
-    Find the update site URL.  E.g. for the [0.7.0 release](https://github.com/OpenLiberty/liberty-tools-eclipse/releases/tag/liberty-tools-0.7.0.202212141445), the URL would be: https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/liberty-tools-eclipse/0.7.0/repository/
+    Find the update site URL for your chosen release from the entry in the Releases list. For example, the URL for the [0.7.0 release](https://github.com/OpenLiberty/liberty-tools-eclipse/releases/tag/liberty-tools-0.7.0.202212141445) is https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/liberty-tools-eclipse/0.7.0/repository/.
 
-    Click on the `Add...` button to open the `Add Repository` view. Specify a name, copy/paste the URL above as the location, and click on the `Add` button. 
+    Click the **Add...** button to open the **Add Repository** view. Specify a name, copy and paste the update site URL as the location, and click the **Add** button. 
```

Comment:
> Specify a name,....

What name? The screenshot shows "Liberty Tools"- is that required or is this a customizable field?

Reply:
addressed in my commit